recommendation/views/images.py

```diff
@@ -0,0 +1,35 @@
+from urllib.parse import urlencode
+
+import requests
+from flask import abort, Blueprint, request, Response, stream_with_context
+
+from recommendation import conf
+
+EMBEDLY_RESIZE = 'https://i.embed.ly/1/display/resize'
+
+
+images = Blueprint('images', __name__)
+
+
+def make_embedly_url(**kwargs):
+    if 'url' not in kwargs:
+        raise RuntimeError()
+    qs = {}
+    for param in ['url', 'width', 'height']:
+        if param in kwargs:
+            qs[param] = kwargs[param][0]
+    qs['key'] = conf.EMBEDLY_API_KEY
```

Comment:
I'm not sure, but maybe we should specify ``animate: false`` and ``compresspng: true`` in the outgoing API call.

Turning off animation makes sure that we don't return animated images, but just the first frame of one. I think in the UI context where we want to show these images, animated images would be a distraction.

compresspng lowers the returned file sizes at the expense of a slightly longer response time from embedly. But since both embedly and we are caching the response, this added latency shouldn't be visible to the end users all too often. And if the cache is effective, most of them will benefit from the smaller image sizes.